queequeg.js

```diff
@@ -1,26 +1,40 @@
-var express = require('express')
-var http = require('http')
-var util = require('util')
-
-var app = express()
+var express = require('express');
+var http = require('http');
+var async = require('async');
+var Singularity = require('singularity-client');
+
+var port = process.env.PORT || 3000,
+    host = process.env.HOST || 'localhost',
+    hookPrefix = "/webook",
```

Comment:
This is the path you prefixed all of the callback urls with. I moved it up here because we need it to setup the routes for the express app and to pass into the client to build the callbacks.